pyart/retrieve/_echo_class_wt.py

```diff
@@ -0,0 +1,313 @@
+"""
+Classification of Precipitation Echoes in Radar Data.
+
+Created on Thu Oct 12 23:12:19 2017
+@author: Bhupendra Raut
+@modifed: 11/19/2023
+@references: 10.1109/TGRS.2020.2965649
+
+.. autosummary::
+    wavelet_reclass
+    label_classes
+    calc_scale_break
+    atwt2d
+"""
+
+
+import numpy as np
+
+
+def wavelet_reclass(
+    grid,
+    refl_field,
+    level,
+    zr_a,
+    zr_b,
+    core_wt_threshold,
+    conv_wt_threshold,
+    scale_break,
+    min_reflectivity,
+    conv_min_refl,
+    conv_core_threshold,
+):
+    """
+    Compute ATWT described as Raut et al (2008) and classify radar echoes using scheme of Raut et al (2020).
+    First, convert dBZ to rain rates using standard Z-R relationship or user given coefficients. This is to
+    transform the normally distributed dBZ to gamma-like distribution, enhancing the structure of the field.
+
+    Parameters:
+    ===========
+    dbz_data: ndarray
+        2D array containing radar data. Last dimension should be levels.
+    res_km: float
```

Comment:
Space between parameters and type, so for example:
res_km: float to res_km : float